tests/layer_tests/tensorflow_tests/test_tf_Reduce.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestReduceOps(CommonTFLayerTest):
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.random.random(inputs_dict[input])
+        return inputs_dict
+
+    def create_reduce_net(self, shape, operation, keep_dims, axis, ir_version):
+        import tensorflow as tf
+        fn_mapping = {'sum': tf.reduce_sum,
+                      'max': tf.reduce_max,
+                      'min': tf.reduce_min,
```

Comment:
it is one that generates `Min` operation? Or do we also need `tf.math.reduce_min`?

Reply:
Yes. `https://www.tensorflow.org/api_docs/python/tf/math/reduce_min` and `https://www.tensorflow.org/api_docs/python/tf/reduce_min` are **aliases** and they produce TF operation with type `Min`.

Refer to the `https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/math_ops.cc#L1046` and `https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/math_ops.py#L2887`